images/windows-hpc-base/build-hpc-base-image.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+
+# Create a directory to build the image content in
+rm -rf build
+BUILD_DIR=build
+
+#trap "rm -rf $BUILD_DIR" EXIT
+
+# Create necessary sub-directorctories
+mkdir -p "$BUILD_DIR/layer/Files/Windows/System32/config"
+
+# Create the files that ProcessBaseLayer on Windows validates when unpacking container images
+touch "$BUILD_DIR/layer/Files/Windows/System32/config/DEFAULT"
+echo "" > "$BUILD_DIR/layer/Files/Windows/System32/config/DEFAULT"
```

Comment:
I think the touch in these lines is superfluous. Maybe drop them?

Reply:
I think I can remove the echo actually.
Let me verify and I'll remove one or the other